hawc/apps/riskofbias/views.py

```diff
@@ -73,7 +73,7 @@ def grouped(metrics: Iterable) -> list[list[models.RiskOfBiasMetric]]:
         )
         context.update(
             metrics=metrics,
-            no_data=metrics == len(metrics) == 0,
+            no_data=len(metrics) == 0,
```

Comment:
I ended up changing this - which fixed the existing message we had I think it was trying to compare `[] == True` which must have been an error